runtime/lua/vim/_defaults.lua

```diff
@@ -227,6 +227,34 @@ do
       vim.lsp.buf.selection_range(-vim.v.count1)
     end, { desc = 'vim.lsp.buf.selection_range(-vim.v.count1)' })
 
+    vim.keymap.set('o', 'an', function()
```

Comment:
the mappings are identical except for the `vim.v.count1` param. just define a `local function` and re-use it